packages/helpers/packages/ArrayHelpers/index.mdx

```diff
@@ -4,8 +4,10 @@ route: /helpers/ArrayHelpers
 menu: 6.Helpers
 ---
 
+
 # ArrayHelpers
 > ✴ Methods that helps us with arrays
+----
```

Comment:
`----` this code makes this part a section, so we got a sub-menu on sidebar with a long name (ex: Methods that helps us to handle os related things) , could we remove this lines? 

 [demo](https://deploy-preview-149--ft-box.netlify.com/helpers/OSHelpers#-methods-that-helps-us-to-handle-os-related-things)

Reply:
yeah, to fix it, instead of disabling sub menus, I'm gonna trim the sub menu names